sanitize_filenames.py

```
@@ -0,0 +1,131 @@
+        return False
+    
+    # Check for trailing spaces or dots
+    if filename.endswith(' ') or filename.endswith('.'):
+        return False
+    
+    # Check if filename contains only allowed characters (alphanumeric, underscore, hyphen, period)
+    allowed_pattern = r'^[a-zA-Z0-9_\-\.]+$'
+    if not re.match(allowed_pattern, filename):
+        return False
+        
+    return True
```

Ruff (0.11.9)

Return the condition `re.match(allowed_pattern, filename)` directly

Replace with `return re.match(allowed_pattern, filename)`

(SIM103)